ui/values/localizable/es.go

```diff
@@ -478,4 +479,8 @@ syncCompTime" = "Est. Sync completion time"
 "selectDexServerToOpen" = "Select the Dex server you would like to open."
 "addDexServer" = "Add dex server"
 "canBuy" = "Can Buy"
+"removeWalletInfo" = "%v Are you sure you want to remove %v %s%v? Enter the name of the wallet below to verify. %v"
+"propNotif" = "Proposal notification"
+"peer" = "Peer"
+"confirmUmixedSpending" = "Confirm to allow spending from unmixed accounts"
```

Comment:
This should be translated to spanish

Reply:
it doesn't necessarily have to, we would have a translator do all non-english translations

Reply:
The idea is to have a place holder for the other languages pending when we do a general sting localization like the https://github.com/planetdecred/godcr/pull/906